books/free-programming-books-es.md

```diff
@@ -202,6 +202,7 @@
 ### JavaScript
 
 * [El Tutorial de JavaScript Moderno](https://es.javascript.info) - Ilya Kantor, Elizabeth Portilla, joaquinelio, Ezequiel Castellanos, et al. (HTML)
+* [Eloquent JavaScript](https://eloquentjs-es.thedojo.mx) - Marijn Haverbeke, `trl.:` Various ([HTML](https://eloquentjs-es.thedojo.mx/00_intro.html), [PDF](https://eloquentjs-es.thedojo.mx/Eloquent_JavaScript.pdf), [EPUB](https://eloquentjs-es.thedojo.mx/Eloquent_JavaScript.epub), [MOBI](https://eloquentjs-es.thedojo.mx/Eloquent_JavaScript.mobi))
```

Comment:
just one link, please

Reply:
Sry, first timer. I changed it so there's only one link for the HTML and left the others for the different formats.

Reply:
one link for the work, you have 4

Reply:
Ok, shouldn't we add a link for each format like this?:
![image](https://user-images.githubusercontent.com/12052430/194615023-d8d581a1-0233-49df-ae32-b3c31b9c024b.png)
[Fortran books on the EN list](https://github.com/EbookFoundation/free-programming-books/blob/main/books/free-programming-books-langs.md#fortran)
At [CONTRIBUTING ](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#formatting) there seems to be several links for multiple formats examples 
![image](https://user-images.githubusercontent.com/12052430/194615604-0e542785-9877-4ec0-9f8d-a57d4a79c4fc.png)
but I'm not sure if it's allowed for all other formats (2 total links) or one for each; as it would be helpful for people wanting to download that specific format w/o having to look for it on the source page.
Thank you.

Reply:
Ok, thank you!